.github/CODEOWNERS

```diff
@@ -1337,7 +1341,8 @@ x-pack/test_serverless/**/test_suites/observability/ai_assistant @elastic/obs-ai
 /x-pack/test/api_integration/services/ml.ts @elastic/ml-ui
 
 # Additional plugins and packages maintained by the ML team.
-/test/examples/response_stream @elastic/ml-ui
+/x-pack/test/functional/services/aiops @elastic/ml-ui
```

Comment:
Could you please group this file with the `/x-pack/test/functional/services/transform/` entry a few lines down.